[New Product] Added antiX Linux

alexkoster opened the pull request:
Adding antiX Linux as a new product! Please feel free to modify this pull request or let me know what changes I can make.

captn3m0 commented on products/antixlinux.md:
```diff
@@ -0,0 +1,60 @@
+title: antiX
+category: os
+releasePolicyLink: https://www.antixforum.com/forums/topic/when-is-end-of-support-for-stable-antix-versions-17-19/#post-26424
+iconSlug: mxlinux
```
```suggestion
iconSlug: NA
```

The antix website doesn't re-use the mx logo, so we shouldn't either.

usta replied:
@alexkoster why did you mark this resolved ? did i miss anything about this one

alexkoster replied:
> @alexkoster why did you mark this resolved ? did i miss anything about this one
@usta My apologies - I think I pushed the wrong button on the mobile app but I went ahead and removed the iconSlug line in the latest update to the [antiX branch](https://github.com/alexkoster/endoflife.date/tree/antix).  Or feel free to let me know or make modifications to keep the iconSlug line but with a value of NA. 

I was working on a commit to the [simple-icons](https://github.com/simple-icons/simple-icons) repository for antiX and Devuan but was having some trouble with scaling and centering the SVG properly.  I hope to request help and make those commits in the future so antiX and Devuan will feature icons.

Thanks again for your time and please let me know if you have any further suggestions for changes or updates.

captn3m0 replied:
Adding `iconSlug: NA` back. We can have a separate PR to add the icon again once the simple-icons repo has them

captn3m0 commented on products/antixlinux.md:
```diff
@@ -0,0 +1,60 @@
+iconSlug: mxlinux
+activeSupportColumn: true
+versionCommand: cat /etc/os-release
+releaseColumn: false
```
```suggestion
releaseColumn: true
```

Considering there is important (and surprising) information below about 22 being the latest release in the 21 release cycle - we should show the latest release in each cycle.

captn3m0 commented on products/antixlinux.md:
```diff
@@ -0,0 +1,60 @@
+-   releaseCycle: "15"
+    codename: "Killah P / Fusion / Berta Cáceres"
+    releaseDate: 2015-06-30
+    latest: "16.3"
```
Couldn't find a source for 16 being part of 15 cycle

alexkoster replied:
@captn3m0 Both are based on Debian 8 "Jessie" and went [EOL at the same time](https://mxlinux.org/blog/mx15-and-mx16-are-end-of-life/). I'm happy to break them out as separate releaseCycles if you'd like.

captn3m0 replied:
Yes please, separate would be better.

alexkoster replied:
@captn3m0 See latest [commit](https://github.com/alexkoster/endoflife.date/commit/469e86b50603e6a867ad6506f8cf57757d72154c) on branch `antix` for separate 15 and 16 release cycles. Thanks for your time reviewing and please let me know if you'd like any further changes prior to merge.

captn3m0 commented on products/antixlinux.md:
```diff
@@ -0,0 +1,60 @@
+
+---
+
+> [antiX](https://antixlinux.com/) is a lightweight desktop-oriented systemd-free GNU/Linux distribution based on Debian's stable branch releases.  AntiX Linux uses the "antiX Magic" IceWM-based desktop to reduce system resource usage.  This allows for antiX to run smoothly on old and new hardware, both from live media and from an installation to disk.
```
```suggestion
> [antiX](https://antixlinux.com/) is a lightweight desktop-oriented systemd-free GNU/Linux distribution based on Debian's stable branch releases. AntiX Linux uses the lightweight "antiX Magic" IceWM-based desktop which allows for antiX to run smoothly on old and new hardware.
```

captn3m0 commented on products/antixlinux.md:
```diff
@@ -0,0 +1,60 @@
+
+> [antiX](https://antixlinux.com/) is a lightweight desktop-oriented systemd-free GNU/Linux distribution based on Debian's stable branch releases.  AntiX Linux uses the "antiX Magic" IceWM-based desktop to reduce system resource usage.  This allows for antiX to run smoothly on old and new hardware, both from live media and from an installation to disk.
+
+There are several releases of antiX supported concurrently.  These typically follow Debian's periods of active development and extended, long-term support, which provides security patches to a limited set of packages.  Often this equates to ~2 years of active support and 2 years of long-term security patch support.
```
```suggestion
There are several releases of antiX supported concurrently.  These typically follow Debian's periods of active development and extended long-term support, which provides security patches to a limited set of packages.  Often this equates to ~2 years of active support and 2 years of long-term security patch support.
```

captn3m0 commented on products/antixlinux.md:
```diff
@@ -0,0 +1,60 @@
+
+There are several releases of antiX supported concurrently.  These typically follow Debian's periods of active development and extended, long-term support, which provides security patches to a limited set of packages.  Often this equates to ~2 years of active support and 2 years of long-term security patch support.
+
+While antiX defaults to using SysVinit as the default init daemon, runit is offered as an alternative.  AntiX officially supports 32-bit non-PAE (i586-compatible), 32-bit PAE (i686-compatible) and 64-bit x86 processors.  Like [MX Linux](https://antixlinux.com) antiX features many custom-developed tools.  These assist with tasks such as package management, bootloader configuration, and persistent  ...
```
```suggestion
AntiX officially supports 32-bit non-PAE (i586-compatible), 32-bit PAE (i686-compatible) and 64-bit x86 processors.
```

A link for this would be nice as well.

usta commented on products/antixlinux.md:
```diff
@@ -0,0 +1,60 @@
+    latestRelaseDate: 2021-05-21
+    support: 2022-09-10
+    eol: 2024-06-30
+    link: https://mxlinux.org/blog/mx-19-4-now-available/
```
wrong link, probably you mixed with mx linux

alexkoster replied:
@usta, good catch! I believe this has been corrected in the latest update to the [antix branch](https://github.com/alexkoster/endoflife.date/tree/antix) but please let me know if you'd like me to make any further changes.

usta commented on products/antixlinux.md:
```diff
@@ -0,0 +1,60 @@
+-   releaseCycle: "19"
+    codename: "Marielle Franco / Hannie Schaft / Manolis Glezos / Grup Yorum"
+    releaseDate: 2019-10-17
+    latest: "19.4"
```
according to its website 19.5 exist : https://antixlinux.com/antix-19-5-point-release-update/

alexkoster replied:
@usta Thanks and good catch! Please see the updated [antix branch](https://github.com/alexkoster/endoflife.date/tree/antix) for this change.  Thanks for your time and please let me know if any further changes are needed.

alexkoster committed:
[antiX] New Product: Added antiX Linux

alexkoster committed:
Merge branch 'endoflife-date:master' into antix

alexkoster committed:
[antix] Update releaseCycle 19 to 19.5 point release

usta commented on products/antixlinux.md:
```diff
@@ -0,0 +1,68 @@
+versionCommand: cat /etc/os-release
+releaseColumn: false
+releaseDateColumn: true
+realseLabel: "__RELEASE_CYCLE__ (__CODENAME__)"
```
@alexkoster typo releaseLabel
Also if you put that long codenames it will look so messy in output

usta replied:
you will understand what i mean in preview after you fix releeaseLabel typo above

alexkoster replied:
@usta Good suggestion and good catch.  I corrected the releaseLabel line and reduced the codenames to a single name in the latest update to the [antix branch](https://github.com/alexkoster/endoflife.date/tree/antix).  Thanks again for your time and please let me know if you'd like me to make any further changes.

alexkoster committed:
[antixlinux] removed iconSlug and reduced codename lengths

alexkoster committed:
[antixlinux] fixed releaseLabel line

alexkoster committed:
Merge branch 'endoflife-date:master' into antix

captn3m0 committed:
Update products/antixlinux.md

captn3m0 committed:
Apply suggestions from code review

Content should not focus on the product, but only the relevant bits of the product (support/security/versioning/lifecycle etc)

captn3m0 committed:
Update antixlinux.md
